### PR TITLE
Addition of DragNETs to Station Armouries

### PR DESCRIPTION
**Pull request by Steelpoint**
:cl: 
add: Addition of two security DragNETs to Deltastations, Omegastations and Metastations armouries.
/:cl:

Currently only Boxstation and Birdstation contain DragNETs as armoury standard issue. This PR simply adds two DragNETs to the above prior mentioned station armouries.

### CODE REVIEW

**Comment by jughu** [comment]
remember when we said these should be ninja only

**Comment by kevinz000** [comment]
Remember when we didn't care and added it to box?

**Comment by kevinz000** [comment]
Also it doesn't compile.

**Comment by Steelpoint** [comment]
How does that work?

**Comment by optimumtact** [comment]
remember when we forced steeliopoint to remove them from box and meta?

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 //MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
-//Model dictionary trimmed on: 31-01-2017 23:17 (UTC)
+"" = (
+UTC)
```
Comment (Jordie0608) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Remove these two lines, issue caused by map merger

**Comment by anconfuzedrock** [comment]
I remember these things being weird as hell and teleporting people randomly explain how they work

**Comment by Cheridan** [comment]
The target is teleported to whatever beacon the teleporter room is set to. It's not random at all, though it does require setup.

**Comment by SweaterKittens** [comment]
I'm not really a fan of giving security even more of a robust arsenal at roundstart. Make people work for their fun toys instead of starting the round with everything they want and being forced to do nothing but wait for valids. If it was selection thing (e.g. the Warden could choose to spawn with the Cycler or the DragNET) I'd be on board.

**Comment by Steelpoint** [comment]
I can see you're point @SweaterKittens however I feel that in most cases a Taser or even a stock Disabler is superior to the DragNET, the DragNET comes into use in edge scenarios where direct attacks on a target are ineffective, such as energy sword or energy shield users or changelings.

I don't personally think the DragNET is a powerful item.

**Comment by Steelpoint** [comment]
Also thanks @Jordie0608 that fixed the issue.

**Comment by SweaterKittens** [comment]
@Steelpoint I don't disagree with you. But just to be clear, I'm not necessarily arguing that it's better than a taser or a disabler - My argument is that having a perfect tool for every situation available to you at roundstart isn't a great way to do things, in my opinion. 

Take Delta Station in it's first iteration: I watched a team of nukeops get absolutely creamed by the security team, because they had such a wide array of weapons that there was no way to properly counterplay or fight back. There were autorifles and the det's revolver for long-range ballistic, shotguns for close range, the Warden's Cycler for more ballistic CQC, Tasers/Disablers, the HOP/HOS/Captain's egun/laser, and a security temperature gun. This is all roundstart equipment that covers just about every sort of threat you could run into. 

I realize that the solutions to this are outside the scope of a simple mapping addition, but I think it's worth discussing. If you want to have an answer to every threat, you should have to make sacrifices, or at least work for it. A system where you could choose between the unique weapons in the armory (tempgun/dragnet/etc), or a point-buy system where you could effectively select your own armory at roundstart would be excellent things to pursue, in my opinion. There just needs to be some give and take beyond having an answer for everything at roundstart.

**Comment by Steelpoint** [comment]
Huh, didn't know the Warden got a neat new gun. Though it seems pretty well balanced as its a auto shotgun that can hold one more shell than a Riot Shotgun, but it can't be sawed off and put into a backpack. 

Your suggestions seem like the 'security points' system Oldman proposed on this thread: https://tgstation13.org/phpBB/viewtopic.php?f=10&t=9376

**Comment by Cheridan** [comment]
> the DragNET comes into use in edge scenarios where direct attacks on a target are ineffective, such as energy sword or energy shield users or changelings.

Yeah, I designed it as an anti-murderbone/anti-saxxing tool. It's not really for general use, but rather a niche weapon to bring out when the situation calls for it.

**Comment by jughu** [comment]
When will it get a better sprite

**Comment by Steelpoint** [comment]
When someone sprites a better one.

I was thinking a cross of a netgun with some techno underbarrel launcher that acts as the teleport shotgun.

**Comment by jughu** [comment]
When everything looks radical as fuck in armouru and then you have those 2 weaponry made in mspaint

**Comment by anconfuzedrock** [comment]
whats the other one

**Comment by Steelpoint** [comment]
The two DragNETs

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -1,5 +1,4 @@
 //MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
-//Model dictionary trimmed on: 31-01-2017 23:17 (UTC)
```
Comment (optimumtact) [inline _maps/map_files/MetaStation/MetaStation.dmm]: can you please reinsert this line using whatever is on the latest master for metastation?

**Comment by Steelpoint** [comment]
I've added the most up to date line back to the metastation file.

**Comment by optimumtact** [comment]
stelioooooooooooo you've come along

**Comment by PKPenguin321** [comment]
>When everything looks radical as fuck in armoury and then you have those 2 weaponry made in mspaint

Truest comment on this whole PR

**Pull request merged**